src/avio.jl

```diff
@@ -631,8 +631,11 @@ if have_avdevice()
         camera_devices = String[]
 
         read_vid_devs = false
-        out, err = readall_stdout_stderr(`$ffmpeg -list_devices true -f $idev -i $idev_name`)
-        buf = length(out) > 0 ? out : err
+        #out, err = readall_stdout_stderr(`$ffmpeg -list_devices true -f $idev -i $idev_name`)
+        #buf = length(out) > 0 ? out : err
+        
+        out = open(`$ffmpeg -list_devices true -f $idev -i $idev_name`, devnull)
+        buf = length(out) > 0 ? out : Base.stderr
```

Comment:
How about trying something like this:

```julia
stdout_buf = IOBuffer()
stderr_buf = IOBuffer()
run(pipeline(ignorestatus(`$ffmpeg -list_devices true -f $idev -i $idev_name`), stdout=stdout_buf, stderr=stderr_buf))
buf = (position(stdout_buf) == 0) ? stdout_buf : stderr_buf
seekstart(buf)

for line in eachline(buf)
   ...
```

There might be a more elegant way to do this, but this seems like it should work.

Reply:
Ok. I can't test at my end right now, but I've made the change and will see how appveyor does 👍